### ISSUE TITLE
Sharing files under MS Windows 10 Mobile 

### ISSUE THREAD

**Issue by MichaelTheWolf** [owncloud/OwncloudUniversal#14]
Switching from Blackberry Q10 to HP Elite x3, suddenly discovered there is no more an opportunity to share files. Am I mistaken, or it's a sad reality?

**Comment by pestotoast** [comment]
Yes, this functionality is not implemented yet. But it will be part of future versions. See #7  and #4
